### ISSUE TITLE
Create colors

### ISSUE THREAD

**Issue by richelbilderbeek** [tresinformal/game#322]
The game uses colors to determine paper-rock-scissors dynamics. The increase the readability/Englishness of our code, we use the colors' names instead of their raw RGB values. For example, we use `create_red_color()` over `color(255,0,0)`. Create the three functions to create the three primary colors: 

```c++
  //#define FIX_ISSUE_322
  #ifdef FIX_ISSUE_322
  // Colors have the correct RGB values
  {
    const color red = create_red_color();
    assert(get_redness(red) == 255);
    assert(get_greenness(red) == 0);
    assert(get_blueness(red) == 0);
  }
  {
    const color green = create_green_color();
    assert(get_redness(green) == 0);
    assert(get_greenness(green) == 255);
    assert(get_blueness(green) == 0);
  }
  {
    const color blue = create_blue_color();
    assert(get_redness(blue) == 0);
    assert(get_greenness(blue) == 0);
    assert(get_blueness(blue) == 255);
  }
  #endif // FIX_ISSUE_322
```

**Comment by TheoPannetier** [comment]
Oops, accidentally solved this as part of #229

**Comment by TheoPannetier** [comment]
solved
